ui/packages/catalog-ui-search/src/main/webapp/react-component/location/common.js

```diff
@@ -64,8 +66,13 @@ const Hemisphere = ({ value, onChange }) => (
   </Group>
 )
 
+const MinimumSpacing = styled.div`
```

Comment:
✏️ This doesn't belong in `location/common.js`

Reply:
I agree but I don't have a better place to put this for now so I'm going to leave it here. Thanks for the feedback though :100: